x/foundation/client/testutil/grpc.go

```diff
@@ -31,7 +30,6 @@ func (s *IntegrationTestSuite) TestGRPCParams() {
 				Params: foundation.Params{
 					FoundationTax: sdk.MustNewDecFromStr("0.2"),
 					CensoredMsgTypeUrls: []string{
-						sdk.MsgTypeURL((*stakingtypes.MsgCreateValidator)(nil)),
```

Comment:
Why does this checking remove?

Reply:
Because, the suite has removed it from the state.